app/views/storage.html

```diff
@@ -22,7 +22,7 @@
           <div class="row">
             <div class="col-md-12">
               <div class="page-header page-header-bleed-right page-header-bleed-left">
-                <div class="pull-right">
+                <div class="pull-right" ng-if="project && 'persistentvolumeclaims' | canI : 'create'">
```

Comment:
Tested it without them and it worked, but just to match other occurrences of it, I will add them.
Thanks !